### ISSUE TITLE
"test" mode, or some way to avoid inviting real users

### ISSUE THREAD

**Issue by simpixelated** [kgish/assembla-to-jira-migration#29]
I just ran step 8 and apparently Atlassian defaults users to active and sends an email invite. I'm not ready for that to happen, so I just want a way to create fake users (like admin+1, admin+2) or at the very least default to inactive so people aren't being notified of my testing.

**Comment by simpixelated** [comment]
Similarly, it would be helpful to have a way to limit the Assembla export. I had assumed `TICKETS_CREATED_ON` did that, but it looks like that only filters after you've exported the data. With over 4 years of data in Assembla, I'm currently looking at 8206 requests in step 2.

**Comment by kgish** [comment]
I agree, will look into implementing this. Anyone out there who can help me?

**Comment by kgish** [comment]
Actually, the API offers an extra flag to prevent sending email notifications, but for some reason this doesn't seem to work for the cloud.
